metadata-ingestion/src/datahub/emitter/rest_emitter.py

```diff
@@ -143,10 +146,31 @@ def build_session(self) -> requests.Session:
         return session
 
 
+@dataclass
+class Chunk:
```

Comment:
since this is an internal dataclass only used in this file

```suggestion
class _Chunk:
```